File: 231119/문자에 따른 명령 2/text-based-commands2.py
N=str(input())

x,y=0,0

dx,dy=[1,0,-1,0],[0,-1,0,1]

dir_num=3
for cmd in N:
    if cmd=='L':
        dir_num=(dir_num-1+4)%4
    elif cmd=='R':
        dir_num=(dir_num+1)%4
    else:
        x,y=x+dx[dir_num],y+dy[dir_num]

print(x,y)

File: 251210/문자에 따른 명령 2/text-based-commands2.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');
const commands = input[0].split('');

// Please Write your code here.
const dx = [1,0,-1,0], dy = [0,-1,0,1];
let x =0, y=0;
let dir=3;
for(let cmd of commands){
    if(cmd==='L'){
        dir = (dir-1+4)%4;
    }else if(cmd==='R'){
        dir = (dir+1)%4;
    }else{
        x+=dx[dir], y+=dy[dir];
    }
}

console.log(x,y)